### PR TITLE
Remove Feature Flag LPS-167536 and LPS-168886

### PR DESCRIPTION
**Pull request by murilostodolni**
https://issues.liferay.com/browse/LPS-170424
https://issues.liferay.com/browse/LPS-172753

~This PR is blocked waiting for https://github.com/liferay-objects/liferay-portal/pull/1757 to be merged.~

### CODE REVIEW

File: modules/apps/list-type/list-type-service/src/main/java/com/liferay/list/type/service/impl/ListTypeEntryLocalServiceImpl.java [hunk 1]
```diff
@@ -178,12 +176,6 @@ private void _validateExternalReferenceCode(
 		String externalReferenceCode, long companyId, long listTypeDefinitionId,
 		long listTypeEntryId) {
 
-		if (Validator.isNull(externalReferenceCode) ||
-			!GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-168886"))) {
-
-			return;
-		}
```
Comment (carolmariaabb) [inline modules/apps/list-type/list-type-service/src/main/java/com/liferay/list/type/service/impl/ListTypeEntryLocalServiceImpl.java:185]: @murilostodolni should we keep the `Validator.isNull(externalReferenceCode)` here?

Comment (murilostodolni) [reply modules/apps/list-type/list-type-service/src/main/java/com/liferay/list/type/service/impl/ListTypeEntryLocalServiceImpl.java]: Yes @carolmariaabb! Just did it. Thank you

**Review by carolmariaabb — APPROVED** [review]
LGTM!

**Comment by murilostodolni** [comment]
ci:test:sf

**Comment by murilostodolni** [comment]
ci:test:sf

**Comment by murilostodolni** [comment]
ci:test:object

**Comment by murilostodolni** [comment]
ci:test:relevant

**Comment by murilostodolni** [comment]
ci:test:relevant

**Comment by murilostodolni** [comment]
ci:test:object

**Comment by murilostodolni** [comment]
Manually forwarding: https://github.com/brianchandotcom/liferay-portal/pull/131074

**Comment by murilostodolni** [comment]
ci:close

**Pull request closed without merging**